_episodes/01-remotes.md

```diff
@@ -39,12 +40,20 @@ $ git clone https://host.com/user/project.git project
 
 # Centralized workflow exercise
 
-Exercise to practice collaborative centralized workflow. On the way to a pull request
-and code review we will meet and discuss a number of typical pitfalls.
+In this exercise, we practice collaborative centralized workflow.
+First, we all **clone** (make a local copy) and try to **push** (send
+code to) the main repository.  We'll see a small problem with that,
+and then make a **pull request** (sending code so that others can
+review and accept later).  We'll discuss how this leads to code review
+and discuss a number of typical pitfalls.
 
+Please don't go too far ahead, because we will learn from problems
+that come up.
 
 ## Before we start
 
+Everyone needs their GitHub account to be added to our central repository.
+
 1. Participants add their usernames to a shared document.
 2. Instructor adds participants as collaborators to this project.
```

Comment:
Maybe we can say in the instructor manuals that for this step, it is preferred that another instructor helps with this.